whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -250,6 +253,19 @@ class ElasticSearch {
         return framed
     }
 
+    private static void setComputedProperties(Document doc) {
+        List<String> isbnValues = doc.getIsbnValues()
+
+        if (isbnValues.size() == 1) {
```

Comment:
Why limit it to documents with a single ISBN?
See e.g. https://libris.kb.se/r82z67c35265vx0/data.jsonld

Reply:
What about ISBNs in `getIsbnHiddenValues()` (`["@graph", 1, "indirectlyIdentifiedBy"]`) ? Do we want to handle them too?

Reply:
Now handling multple ISBN values and generating ISBN10/13 for indirectlyIdentifiedBy.